server/config/.env

```diff
@@ -32,4 +32,6 @@ export APPLICATIONS_DEVELOPER_SERVICE_FUTURE_BASE_URL=${APPLICATIONS_API_FUTURE_
 export APPLICATIONS_DEVELOPER_SERVICE_FUTURE_API_KEY=
 export APPLICATIONS_DEVELOPER_SERVICE_FUTURE_USERNAME=
 export APPLICATIONS_DEVELOPER_SERVICE_FUTURE_PASSWORD=
+
```

Comment:
sorry I think this extra line was my fault

Reply:
Its not an unreasonable way to split up the variables, happy to leave it if you think its more readable, or remove it. No preferences either way.

Reply:
I don't mind , I suppose it does make it more readable so I guess there is no harm in leaving it